docs/user/openstack/ovs-dpdk.md

```diff
@@ -0,0 +1,248 @@
+# Installing a cluster on OpenStack that supports DPDK-connected compute machines
+
+If the underlying OpenStack deployment has Open vSwitch with Data Plane Development Kit (OVS-DPDK) support enabled,
+an OpenShift cluster can take advantage of the feature by providing access to [poll mode drivers][1].
+
+## Pre-requisites
+
+The following steps might be required to be checked before starting the deployment of OpenShift.
+
+### Selecting hugepages size and CPU policy for OpenShift virtual machines
+
+You require hugepages for every OCP cluster virtual machine running on the hosts with OVS-DPDK.
+If hugepages are not present in the flavor, the machines will get created but the network interface won't work.
+You must check with your OpenStack administrator the size and number of available hugepages which can be used per node.
+
+This may be accomplished in two ways:
+
+- The flavors used for the OCP are defined with appropriate values. For example:
+
+    ``` bash
+    openstack flavor create m1.xlarge.nfv --ram 16384 --disk 40 --vcpus 8
+    openstack flavor set --property hw:cpu_policy=dedicated --property hw:mem_page_size=large m1.xlarge.nfv
+    ```
+
+    This method requires administrative access the OpenStack deployment because flavors need to be created and modified.
+
+- The image properties maybe be defined in the install-config.yaml using [platform.openstack.clusterOSImageProperties](customization.md).
+  For example:
+
+    ``` yaml
+    platform:
+      openstack:
+         clusterOSImageProperties:
+            - hw_cpu_policy: dedicated
+            - hw_mem_page_size: 1GB
```

Comment:
Turns out this can be a string like `large` rather than a specific value. Might better to use this so it aligns with the above?